AzFilesHybrid/AzFilesHybrid.psm1

```diff
@@ -183,9 +183,9 @@ function Get-IsDomainJoined {
         "Windows" {
             $computer = Get-CimInstance -ClassName "win32_computersystem"
             if ($computer.PartOfDomain) {
-                Write-Verbose -Message "Session is running in a domain-joined environment."
+                Write-Verbose -Verbose -Message "Session is running in a domain-joined environment."

```

Comment:
I added -Verbose because of task #3 "CheckADObject check doesn't have verbose log". I thought the customer asks for the verbose logging by default. But I guess it is fine to revert this change, since the customer can always add -Verbose when calling the cmdlet.

Reply:
I see - I think I should have better described it in the work item.  Sorry about that.

The issue was in the code snippet:

 if (!$filterIsPresent -or $Filter -match "CheckADObject")
        {
            Debug-AzStorageAccountADObject -storageaccountName $StorageAccountName -ResourceGroupName $ResourceGroupName;
        }

There was no verbose log to indicate the status of this check.
Let's revert the "-Verbose' in Write-Verbose but keep any additional verbose traces you added for this.